### ISSUE TITLE
[Feature Request]:不支持Neural Structured Learning

### ISSUE THREAD

**Issue by xdaker** [SciSharp/TensorFlow.NET#1301]
### Background and Feature Description

不支持使用结构化信号进行训练，希望能添加，谢谢了，还有最新版（0.150.0）版本不支持GPU 。希望能改进。https://tensorflow.google.cn/neural_structured_learning?hl=zh-cn

### Alternatives

# Create a base model -- sequential, functional, or subclass.
model = tf.keras.Sequential([
    tf.keras.Input((28, 28), name='feature'),
    tf.keras.layers.Flatten(),
    tf.keras.layers.Dense(128, activation=tf.nn.relu),
    tf.keras.layers.Dense(10, activation=tf.nn.softmax)
])

# Wrap the model with adversarial regularization.
adv_config = nsl.configs.make_adv_reg_config(multiplier=0.2, adv_step_size=0.05)
adv_model = nsl.keras.AdversarialRegularization(model, adv_config=adv_config)
